tests/tracer/test_trace_utils.py

```diff
@@ -160,3 +161,17 @@ def test_set_http_meta_custom_errors(mock_log, span, int_config, error_codes, st
         mock_log.exception.assert_called_once_with(*log_call)
     else:
         mock_log.exception.assert_not_called()
+
+
+def test_activate_distributed_tracing_context(int_config):
+    pin = Pin()
+    int_config.myint["distributed_tracing_enabled"] = True
+    headers = {
+        HTTP_HEADER_PARENT_ID: "12345",
+        HTTP_HEADER_TRACE_ID: "678910",
+    }
+    trace_utils.activate_distributed_tracing_context(pin, int_config.myint, headers)
+    context = pin.tracer.context_provider.active()
+
+    assert context._parent_trace_id == 678910
```

Comment:
```suggestion
    assert context.trace_id == 678910
```

Reply:
Ahh good catch, I only saw the private attributes 😨